File: common/protcol/c#/caller/gm_centercaller.cs
/*this caller file is codegen by juggle for c#*/
using System;
using System.Collections;
using System.IO;

namespace caller
{
    public class gm_center : juggle.Icaller 
    {
        public gm_center(juggle.Ichannel _ch) : base(_ch)
        {
            module_name = "gm_center";
        }

        public void confirm_gm(String argv0)
        {
            ArrayList _argv = new ArrayList();
            _argv.Add(argv0);
            call_module_method("confirm_gm", _argv);
        }

        public void close_clutter(String argv0)
        {
            ArrayList _argv = new ArrayList();
            _argv.Add(argv0);
            call_module_method("close_clutter", _argv);
        }

        public void close_zone(String argv0,Int64 argv1)
        {
            ArrayList _argv = new ArrayList();
            _argv.Add(argv0);
            _argv.Add(argv1);
            call_module_method("close_zone", _argv);
        }

        public void reload(String argv0,String argv1)
        {
            ArrayList _argv = new ArrayList();
            _argv.Add(argv0);
            _argv.Add(argv1);
            call_module_method("reload", _argv);
        }

    }
}

File: protcol/js/caller/gm_centercaller.js
/*this caller file is codegen by juggle for js*/
function gm_center_caller(ch){
    Icaller.call(this, "gm_center", ch);

    this.confirm_gm = function( argv0){
        var _argv = [argv0];
        this.call_module_method.call(this, "confirm_gm", _argv);
    }

    this.close_clutter = function( argv0){
        var _argv = [argv0];
        this.call_module_method.call(this, "close_clutter", _argv);
    }

    this.close_zone = function( argv0, argv1){
        var _argv = [argv0,argv1];
        this.call_module_method.call(this, "close_zone", _argv);
    }

    this.reload = function( argv0, argv1){
        var _argv = [argv0,argv1];
        this.call_module_method.call(this, "reload", _argv);
    }

}
(function(){
    var Super = function(){};
    Super.prototype = Icaller.prototype;
    gm_center_caller.prototype = new Super();
})();
gm_center_caller.prototype.constructor = gm_center_caller;

